web/src/api/http-client.js

```diff
@@ -29,8 +28,7 @@ httpClient.interceptors.response.use(null, async (error) => {
   // Bounce the user if they hit a login required error when trying to access a protected route
   // It would probably be better to move this code to a route guard or something?
   if (error?.error === "login_required") {
-    const auth0 = await getInstance()
-    auth0.loginWithRedirect({ appState: { targetUrl: window.location.pathname } })
```

Comment:
I'm not sure about this change, maybe we should redirect the user to the "UnauthorizedPage", or re-add the code I removed.

I removed this because it was really hard to debug why you were being redirected.